files/fare_periods_ft.md

```diff
@@ -16,7 +16,6 @@ File MUST contain the following attributes:
 Required Attributes	| Description										
 ----------			| -------------		
 `fare_id`			| An ID that links to `fare_id` in `fare_rules.txt`.  
-`fare_class`		| Contains the name of the `fare_class` that links to the same attribute in `routes_ft.txt`. 
```

Comment:
I don't think you need to issue a new pull request; if you push a commit to the same branch (stefancoe:master), this pull request gets updated.